wcp/heartbeat.tex

```diff
@@ -16,8 +17,19 @@
 			\item \If $\ct_{i} \neq \maxCt_{i}$ \Then $\ct_{i + 1} = 1 + \ct_{i}$
 			\item \If $\ct_{i} =    \maxCt_{i}$ \Then $\wcpStamp_{i + 1} = 1 + \wcpStamp_{i}$
 		\end{enumerate}
+	\label{wcp: heartbeat: smallness of ct max}
 	\item $\ct_{i} \neq \llarge$
+	\label{wcp: heartbeat: determinism of ct max}
+	\item \If $\ct_{i} = 1 $ \Then: 
+	\begin{enumerate}
+			\item \Or $\byteCol{1}_{i-1} \neq 0$
+			\item \Or $\byteCol{2}_{i-1} \neq 0$
+			\item \Or $\byteCol{3}_{i-1} \neq 0$
+			\item \Or $\byteCol{4}_{i-1} \neq 0$
+	\end{enumerate}	
 	\item $\ct_{N} = \maxCt_{N}$.
 \end{enumerate}
-\saNote{} We don't explicitly impose anything on the value of $\maxCt_i$ when $\vli_i \equiv 1$.
-But the above forces us to have at all times $0 \leq \ct_{i} \leq \maxCt_{i} < \llarge$, at the very least whenever $\wcpStamp_{i} \neq 0$ if the implementation omits the constraints marked with $(\trash)$.
+\saNote{} When $\vli_i \equiv 1$, the value of $\maxCt_i$ is imposed:
```

Comment:
There's a problem here in the presentation.  Its not formatted as a bullet point list which I'm assuming is what you intended.  Its actually very confusing and only by looking at the latex source could I understand what you meant.